### PR TITLE
Add mailbox to init flow

### PR DESCRIPTION
**Pull request by dmerrill6**

### CODE REVIEW

**Review by jsonsivar — APPROVED** [review]
LGTM

File: core/textile/client.go [hunk 1]
```diff
@@ -197,6 +199,12 @@ func (tc *textileClient) checkHubConnection(ctx context.Context) error {
 			return err
 		}
 		tc.mb = mailbox
+
+		if err := tc.listenForMessages(hubctx); err != nil {
+			tc.isConnectedToHub = false
```
Comment (maurycy) [inline core/textile/client.go:204]: @dmerrill6 

Do you think we need any locking here?

**Review by maurycy — APPROVED** [review]
LGTM. 🚀

File: core/textile/client.go (hunk 1 above)
Comment (dmerrill6) [reply core/textile/client.go:204]: Will merge to unblock the FE and we can iterate if we find issues

Comment (maurycy) [reply core/textile/client.go:204]: @dmerrill6 

Yes, let's discuss this later.

**Pull request merged**